docs/source/deduplication.rst

```diff
@@ -0,0 +1,42 @@
+System Deduplication and Merging
```

Comment:
Global: I learned to type on a manual typewriter, so sometimes I do the old double-space after periods thing too out of muscle memory/habit. However, do a search & replace and use the electronic standard spacing, which is single space.